bidict/_orderedbase.py

```diff
@@ -98,7 +98,7 @@ def new_last_node(self) -> Node:
         """Create and return a new terminal node."""
         old_last = self.prv
         new_last = Node(old_last, self)
-        old_last.nxt = self.prv = new_last
+        old_last.nxt = self.prv = new_last  # flagged by pyright
```

Comment:
pyright (but not mypy) is flagging these lines, as follows:
```console
  /Users/jab/src/bidict/bidict/_orderedbase.py:61:14 - error: "prv" is not specified in __slots__ (reportGeneralTypeIssues)
  /Users/jab/src/bidict/bidict/_orderedbase.py:101:29 - error: "prv" is not specified in __slots__ (reportGeneralTypeIssues)
```
1. Is this a bug in pyright and/or mypy?
2. What is the best way to appease pyright here?
    1. Again, I can't add a # type: ignore here without causing mypy to fail with `error: Unused "type: ignore" comment`, and wish I at least had the option of doing `# pyright: ignore` on a per-line basis instead.
    2. I think the "reportGeneralTypeIssues" pyright check that is flagging this code is so broad, and can catch too many actual issues, to disable it globally.

Reply:
This is a bug in pyright. It shouldn't be complaining about `prv` here because it's a class variable descriptor, not an instance variable.

I've filed the following bug: https://github.com/microsoft/pyright/issues/3172

Reply:
Thanks for filing that bug! Will follow it.

In the meantime, do you have any suggestions for a workaround here? Ideally something fairly minimal?

And is this another case where something like `# pyright: ignore` would be the most user-friendly workaround, if it were available?

Reply:
I typically publish pyright every Tuesday and Friday, so if you can wait for another 24 hours, this will be fixed.

Reply:
Wow, that’s an impressive release schedule!

I can certainly wait 24 hours for this particular bugfix. My suggestion was more for future bugs we haven’t even discovered yet.

Reply:
Started [a new thread](https://github.com/jab/bidict/pull/242#discussion_r825314856) for the remaining pyright bug here. Marking this thread as resolved.